pkg/internal/approver/allowed/validation.go

```diff
@@ -76,14 +76,32 @@ func (a allowed) Validate(_ context.Context, policy *policyapi.CertificateReques
 	}
 
 	for _, stringSlice := range stringSlices {
-		if stringSlice.slice != nil && stringSlice.slice.Required != nil && *stringSlice.slice.Required && stringSlice.slice.Values == nil {
-			el = append(el, field.Required(stringSlice.path.Child("values"), "values must be defined if required field"))
+		if stringSlice.slice != nil {
+			if stringSlice.slice.Required != nil && *stringSlice.slice.Required {
+				if stringSlice.slice.Values == nil && len(stringSlice.slice.Validations) == 0 {
+					el = append(el, field.Required(stringSlice.path.Child("values"), "'values' or 'validations' must be defined if field is 'required'"))
```

Comment:
A user can specify `values` **AND** `validations`. Does this error message express this as it stands now, or do we have an even better suggestion? 🤔

Reply:
maybe or/and could work, but I think the current message is also correct (the issue in this case is that at least one of the two must be specified)

Reply:
@jsoref Can you help here? :-)

Reply:
"at least one of ..."?

Reply:
Updated.